dlp/snippets/jobs/jobs_test.go

```diff
@@ -34,25 +34,25 @@ func setupPubSub(projectID, topic, sub string) (*pubsub.Subscription, error) {
 	ctx := context.Background()
 	client, err := pubsub.NewClient(ctx, projectID)
 	if err != nil {
-		return nil, fmt.Errorf("pubsub.NewClient: %w", err)
+		return nil, fmt.Errorf("pubsub.NewClient: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.